src/murmur/Meta.cpp

```diff
@@ -50,7 +50,7 @@ MetaParams::MetaParams() {
 	qsPassword                 = QString();
 	usPort                     = DEFAULT_MUMBLE_PORT;
 	iTimeout                   = 30;
-	iMaxBandwidth              = 72000;
+	iMaxBandwidth              = 558000;
```

Comment:
There should be a comment explaining where that number is coming from.

Also does this then mean that by default every client is using this? Or is it only used to determine the maximum possible value in the UI? :thinking:

Reply:
It is the maximum possible value that a client can use to transmit if configured to do so. The GUI configuration limits this to a maximum of 240000. By default the client suggests values much lower than this: https://github.com/mumble-voip/mumble/blob/1455b3969879516d8b0865efec742a0b8b9f4bd0/src/mumble/AudioWizard.cpp#L690-L712

Reply:
Added a comment.

Reply:
Okay if it only increases the maximum without also increasing the default, that's fine :+1: